components/Common/LanguageDropDown/index.module.css

```diff
@@ -0,0 +1,39 @@
+.iconWrapper {
+  @apply h-9
+	w-9
+	rounded-md 
+	bg-neutral-100 
+	p-2 
+	text-neutral-700
+	dark:bg-neutral-900
+	dark:text-neutral-300;
+}
+
+.dropDownContent {
+  @apply max-h-80
+	w-[184px]
```

Comment:
Is this manual value needed? Wouldn't something like w-20 or larger work here? Could you try out?

Reply:
The width of the component in Figma is `184px`. Tailwind does not have a class that exactly fit this value. The 2 closest one to this are `w-44 = 176px` and `w-48 = 192px`.

![image](https://github.com/nodejs/nodejs.org/assets/66557682/eb65355d-e1d1-4003-9498-6ad1df490bcc)